test/api_compatibility.js

```diff
@@ -104,30 +87,66 @@ module.exports = function apiCompatibilityChecks(_createTracer) {
         });
 
         describe('SpanContext', function() {
-            var tracer = createTracer();
-            var span = tracer.startSpan('test_span');
-            var spanContext = span.context();
-
             it('should be a class', function() {
+                var spanContext = span.context();
                 expect(spanContext).to.be.an('object');
             });
         });
 
-
         describe('Reference', function() {
-            var tracer = createTracer();
-            var span = tracer.startSpan('test_span');
-            var ref = new opentracing.Reference(opentracing.REFERENCE_CHILD_OF, span.context());
 
             it('should be a class', function() {
+                var ref = new opentracing.Reference(opentracing.REFERENCE_CHILD_OF, span.context());
```

Comment:
this looks like it should be moved to the other test file you have, for opentracing lib itself. It's not checking compatibility of external tracer implementation.